### PR TITLE
Remove Jade, Node, and Browserify

### PR DESCRIPTION
**Pull request by leahciMic**
This removes the dependencies on Jade, Node, and Browserify and makes it a simple vanilla js sample.

This also makes the JS sample work in Internet Explorer 9+, and Firefox.

Jira: https://jira.tokbox.com/browse/OPENTOK-24034

### CODE REVIEW

File: js-sample/app.js [hunk 1]
```diff
@@ -0,0 +1,304 @@
+var API_KEY = '100'; // your API_KEY
+var SESSION_ID = '1_MX4xMDB-fjE0MzY0ODg2NDA5NjZ-aVFLMHlvbVZoelRWZDdqMWVKRk9UeGh4fn4'; // your SESSION_ID
+var TOKEN = 'T1==cGFydG5lcl9pZD0xMDAmc2lnPTMxN2ViNmU5ZDhhODI5Mzg1MzYzMTdlYzkwZDNmYzE5ZDA4MmQwYTE6c2Vzc2lvbl9pZD0xX01YNHhNREItZmpFME16WTBPRGcyTkRBNU5qWi1hVkZMTUhsdmJWWm9lbFJXWkRkcU1XVktSazlVZUdoNGZuNCZjcmVhdGVfdGltZT0xNDM2NDg4NjQyJm5vbmNlPTI2Mjgmcm9sZT1tb2RlcmF0b3Imc2Vzc2lvbklkPTFfTVg0eE1EQi1makUwTXpZME9EZzJOREE1TmpaLWFWRkxNSGx2YlZab2VsUldaRGRxTVdWS1JrOVVlR2g0Zm40'; // your TOKEN
+var TEST_TIMEOUT_MS = 15000; // 15 seconds
```
Comment (jeffswartz) [inline js-sample/app.js:4]: Please remove these hardcoded values, and replace them with empty strings:

``` /**
* Replace these with your OpenTok API key, a session ID for a routed OpenTok session,
* and a token that has the publish role:
*/
var API_KEY = '';
var SESSION_ID = '';
var TOKEN = '';
```

File: js-sample/.jshintrc [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "browserify": true,
```
Comment (jeffswartz) [inline js-sample/.jshintrc]: Should this line be removed? I thought we are not using browserify.

File: js-sample/app.js [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+
+var publisherEl = document.createElement('div');
+var subscriberEl = document.createElement('div');
+var session;
+var publisher;
+var subscriber;
+var statusContainerEl;
+var statusMessageEl;
+var statusIconEl;
+
```
Comment (jeffswartz) [inline js-sample/app.js]: I recommend putting these variable declarations at the top of the file, along with the testStreamingCapability function and other code most closely related to the stream test. I'd move the helper functions (e.g., pluck(), sum(), etc.) to the end of the file.

Comment (leahciMic) [inline js-sample/app.js]: done!

File: js-sample/app.js (hunk 1 above)
Comment (leahciMic) [inline js-sample/app.js:4]: done!

File: js-sample/.jshintrc (hunk 2 above)
Comment (leahciMic) [inline js-sample/.jshintrc]: Done!

**Pull request merged**